packages/composites/the-graph/src/methods/prices/dex/uniswap.ts

```diff
@@ -1,8 +1,8 @@
-import { DexSubgraph, GraphqlAdapterRequest, TokenInformation } from "../../../types"
+import { DexUniswapSubgraph, GraphqlAdapterRequest, TokenInformation } from "../../../types"
 import { fetchFromGraphqlAdapter } from "../dataProvider"
 import { getPairQuery, getTokenQuery } from "./graphqlQueries"
 
-export class UniswapSubgraph implements DexSubgraph {
+export class UniswapSubgraph implements DexUniswapSubgraph {
```

Comment:
I think that we can remove the `DexUniswapSubgraph` interface as we are no longer going to have a common interface for all Dex subgraphs.